client/deposits/details/style.scss

```diff
@@ -37,6 +37,53 @@
 			font-size: 20px;
 			line-height: 28px;
 		}
+
+		.wcpay-summary__item-detail {
+			color: $dark-gray-500;
+			word-wrap: break-word;
+
+			.woopayments-payout-bank-reference-id {
+				font-family: monospace;
+			}
+
+			.woopayments-payout-bank-reference-id-copy-button {
+				line-height: 1.2em;
+				display: inline-flex;
+				background: transparent;
+				border: none;
+				border-radius: 0;
+				vertical-align: middle;
+				font-weight: normal;
+				cursor: pointer;
+				color: inherit;
+				margin-left: 2px;
+				align-items: center;
+
+				i {
+					display: block;
+					width: 1.2em;
+					height: 1.2em;
+					mask-image: url( 'assets/images/icons/copy.svg?asset' );
+					mask-size: contain;
+					mask-repeat: no-repeat;
+					mask-position: center;
+					background-color: currentColor;
+
+					&:hover {
+						opacity: 0.7;
+					}
+
+					&:active {
+						transform: scale( 0.9 );
+					}
+				}
+
+				&.state--copied i {
+					mask-image: url( 'assets/images/icons/check-green.svg?asset' );
+					background-color: $studio-green-50;
+				}
```

Comment:
There's a lot of css (and some React behaviour) for this little copy-to-clipboard button. Could we package it into a little component? The button is nicely general-purpose so is a good candidate for an abstraction.

Reply:
Great suggestion! I have moved the button to a separate component via [0d26f76](https://github.com/Automattic/woocommerce-payments/pull/9886/commits/0d26f7634a87f47cf6945b0b35abebcd21c98160)

and 

[5c3c6aa](https://github.com/Automattic/woocommerce-payments/pull/9886/commits/5c3c6aad42ebea90934dadb112601443ea2e45a1)

This will be certainly useful with the new design. Once it is approved, I can also suggest it to be used in the checkout from where I originally borrowed :D